index.html

```diff
@@ -2297,10 +2297,10 @@ <h1>Core Representations</h1>
 
     <p>
 Producers MUST indicate which representation of a document has been used via a
-media type in the document's metadata. Consumers MUST determine which
-representation a document is in via the <code>content-type</code> DID resolver
-metadata field. (See <a href="#did-resolution"></a>). Consumers MUST NOT
-determine the representation of a document through its content alone.
+media type in the document's metadata. Consumers MUST determine the
```

Comment:
This won't work for files on disk that have a .jsonld property or a .did extension. In both of those cases, you can derive the representation from the file extension and the content in the file.

Reply:
That language was already there, this change is just a grammar/readability fix.